cfpb/cfgov-refresh
# Migrate custom features to AnswerPage answer field rich text editor

Migrate existing customizations (tip formatter, HTML editor) from the answer rich text field on the Answer model to the analogous field on the AnswerPage.

## Changes

- Remove URL check from `ask_cfpb_tips` and `html_editor` plugins
- Register tip formatter and html editor plugins as features
- Apply tip formatter and html editor features to answer rich text editor on AnswerPage
- Migrate help text from Answer field to AnswerPage field

## Testing

1. Check out branch
2. Run migration
3. Verify that tip formatting and HTML edit view are available on AnswerPage answer field rich text editor and work as expected

## Screenshots
<img width="1221" alt="screen shot 2019-02-19 at 3 23 35 pm" src="https://user-images.githubusercontent.com/778171/53054769-52a8b580-345a-11e9-9f25-0643202b60f7.png">


## Notes

- This should be a short-term fix to allow editors to edit tips and view answer content's HTML. In the longer term, we hope to remove the HTML edit view and handle tip creation via an answer StreamField.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
